### PR TITLE
Application Gateway ARM command-lets

### PR DESCRIPTION
**Pull request by kagamsft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kagamsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kaga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by DeepakRajendranMsft** [comment]
https://github.com/Azure/azure-sdk-for-node/pull/1460 has to be checked in before this is merged. else the network version will roll back

File: ChangeLog.txt [hunk 1]
```diff
@@ -1,4 +1,64 @@
 ﻿2015.07.16 version 0.9.5
+* Azure Network Resource Provider cmdlets
+	* Added Application Gateway cmdlets
```
Comment (hovsepm) [inline ChangeLog.txt]: please fix indentation to 4 spaces on new subsections.

File: src/AzurePowershell.sln [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+VisualStudioVersion = 12.0.21005.1
```
Comment (hovsepm) [inline src/AzurePowershell.sln]: revert this line.

File: src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.ps1 [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+.SYNOPSIS
+Application gateway tests
+#>
+function Test-ApplicationGatewayCRUD
+{
+    # Setup	
+
+	$rgname = Get-ResourceGroupName
+    $rname = Get-ResourceName
+    $domainNameLabel = Get-ResourceName
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.ps1]: fix indentation in this file

**Comment by markcowl** [comment]
@kagamsft There are merge conflicts, please resolve

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Pull request merged**